.github/ISSUE_TEMPLATE/--bug-report.md

```diff
@@ -0,0 +1,36 @@
+---
+name: "🐛Bug report"
+about: Report a problem with Spectrum CSS
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+## Description
+<!-- A clear and concise description of what the bug is. -->
+
+
+## Steps to reproduce
+
```

Comment:
maybe something like URL?
Those will be very useful.

Reply:
Agreed.  That's what I meant with `Go to '...'`, I will make that more clear

Reply:
you could pre-fill a URL in markdown there `[URL](https://adobe.com)` and make it even easier.